torchtune/utils/_checkpointing/_checkpointer_utils.py

```diff
@@ -24,6 +24,7 @@ class ModelType(Enum):
     LLAMA3 = "llama3"
     MISTRAL = "mistral"
     PHI3_MINI = "phi3_mini"
+    MISTRAL_REWARD = "mistral_reward"
```

Comment:
maybe this could be mistral_classifier?

Reply:
I think ```mistral_reward``` makes sense unless you think there will be classifier checkpoints we'll need to load?

Reply:
I'll leave as-is for now, we can always generalise later if there's use cases.